src/Ocelot/Requester/HttpClientHttpRequester.cs

```diff
@@ -32,7 +32,17 @@ public class HttpClientHttpRequester : IHttpRequester
 
             try
             {
-                var response = await httpClient.SendAsync(context.DownstreamRequest.ToHttpRequestMessage());
+                var message = context.DownstreamRequest.ToHttpRequestMessage();
+
+                // Remove Content from the message if not put or post or the http client will choke
```

Comment:
should this be in a preprocessor directive if the problem only exists in .net 461? You can read more about them [here](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if)